include/deal.II/numerics/vector_tools_rhs.templates.h

```diff
@@ -378,7 +364,7 @@ namespace VectorTools
       {
```

Comment:
You need to get rid of the now-unused `cell` and `endc` defined on lines 359-361.

Reply:
Thanks for this hint. Now it is fixed.